src/cinder/audio/linux/FileAudioLoader.cpp

```diff
@@ -609,8 +612,7 @@ void SourceFileAudioLoader::performSeek( size_t readPositionFrames )
 
 std::vector<std::string> SourceFileAudioLoader::getSupportedExtensions()
 {
-	std::vector<std::string> result;
-	return result;
+	return { "mp3" };
```

Comment:
We should add this, but also might as well include file formats that libsndfile supports (https://libsndfile.github.io/libsndfile/formats.html)